libafl_qemu/libafl_qemu_build/src/lib.rs

```diff
@@ -25,6 +30,73 @@ pub fn build_with_bindings(
         .expect("Faield to write to the bindings file");
 }
 
+// For bindgen, the llvm version must be >= of the rust llvm version
+fn find_llvm_config() -> Result<String, String> {
+    if let Ok(var) = env::var("LLVM_CONFIG") {
+        return Ok(var);
+    }
+
+    let rustc_llvm_ver = find_rustc_llvm_version().unwrap();
+    for version in (rustc_llvm_ver..=LLVM_VERSION_MAX).rev() {
+        let llvm_config_name: String = format!("llvm-config-{version}");
+        if which(&llvm_config_name).is_ok() {
+            return Ok(llvm_config_name);
+        }
+    }
+
+    if which("llvm-config").is_ok() {
+        if let Some(ver) = find_llvm_version("llvm-config".to_owned()) {
+            if ver >= rustc_llvm_ver {
+                return Ok("llvm-config".to_owned());
+            }
+        }
+    }
+
+    Err("could not find llvm-config".to_owned())
+}
+
+fn exec_llvm_config(llvm_config: String, args: &[&str]) -> String {
+    match Command::new(llvm_config).args(args).output() {
+        Ok(output) => String::from_utf8(output.stdout)
+            .expect("Unexpected llvm-config output")
+            .trim()
+            .to_string(),
+        Err(e) => panic!("Could not execute llvm-config: {e}"),
+    }
+}
+
+fn find_llvm_version(llvm_config: String) -> Option<i32> {
```

Comment:
Could we do that in targets or somewhere centralized? I think we need it in multiple places, right?

Reply:
It is similar to the code in libafl cc but different

Reply:
also, this thing runs always in build.rs, we may want a centralized build.rs lib maybe but not in libafl_targets for sure